src/plugins/workflow_designer/src/library/ConvertFilesFormatWorker.cpp

```diff
@@ -101,9 +101,9 @@ QVariantMap getFormatsMap(MapType mapType) {
         }
         if (format->checkFlags(DocumentFormatFlag_SupportWriting) || (BOOLEANS == mapType)) {
             if (BOOLEANS == mapType) {
-                result[fid] = false;
+                result[AppContext::getDocumentFormatRegistry()->getFormatById(fid)->getFormatName()] = false;
```

Comment:
Dirty code that will be hard to debug. Could you please cache the value into a local var.
Same for all other files in the PR where you use this long expressions.